fw/procfs.h

```diff
@@ -89,11 +90,17 @@ typedef struct {
 	u64	misses;
 } TfwCacheStat;
 
+/*
+ * @hm	- Total responses for each HTTP code specified in 'health_stat'.
+ * 	  Accounts for all responses, whether served from the backend or the
+ *	  cache.
+ */
 typedef struct {
 	TfwSsStat	ss;
 	TfwClntStat	clnt;
 	TfwSrvStat	serv;
 	TfwCacheStat	cache;
+	TfwHMStats	*hm;
```

Comment:
Personally, I would make it a structure member here, not a pointer to structure, in unison with all other members of the structure. The per cpu allocation of the array for stats per each response code would be simpler and cleaner, in my opinion.